pkgs/applications/video/mpv/default.nix

```diff
@@ -93,6 +94,15 @@ in stdenv.mkDerivation (self: {
     sha256 = "sha256-CoYTX9hgxLo72YdMoa0sEywg4kybHbFsypHk1rCM6tM=";
   };
 
+  patches = [
+    (fetchpatch {
+      # fixes EDL error on youtube DASH streams (mpv #11392)
+      # to be removed on next release
+      url = "https://patch-diff.githubusercontent.com/raw/mpv-player/mpv/pull/11398.patch";
```

Comment:
I don't remember precisely why - maybe it was some advice from @SuperSandro2000, but hashed urls are better suited than github-numbered ones.

But I will not pursue this now.

Reply:
https://github.com/mpv-player/mpv/commit/362256edbc4f95c63e69c1fa8c8dce9cc6c44288.patch

Reply:
I'm happy to change it, but my instinct is the opposite. Using the github-numbered url makes it easier to trace what PR the patch originates from.

Reply:
If the PR is not merged they cannot be used because force pushes will change that hash.